identity/aziot-identityd/src/identity.rs

```diff
@@ -266,6 +344,20 @@ impl IdentityManager {
                 let identities = response
```

Comment:
You don't need to fall back to calling `get_module_backup` if the `client.get_modules()` on line 342 fails?

Reply:
Only single identity read operations are supported offline. All other APIs fail when the device isn't connected.